activerecord/lib/active_record/connection_adapters/column.rb

```diff
@@ -13,7 +13,7 @@ class Column
 
       # Instantiates a new column in the table.
       #
-      # +name+ is the column's name, such as <tt>supplier_id</tt> in <tt>supplier_id bigint</tt>.
+      # +name+ is the column's name, such as +supplier_id+ in <tt>supplier_id bigint</tt>.
       # +default+ is the type-casted default value, such as +new+ in <tt>sales_stage varchar(20) default 'new'</tt>.
```

Comment:
```suggestion
      # +default+ is the type-casted default value, such as +new+ in +sales_stage varchar(20) default 'new'+.
```
Or does it not work because of the spaces?